src/Symfony/Bundle/SecurityBundle/Resources/config/security_authenticator_access_token.php

```diff
@@ -26,9 +26,9 @@
         ->set('security.authenticator.access_token', AccessTokenAuthenticator::class)
             ->abstract()
             ->args([
-                abstract_arg('user provider'),
```

Comment:
this change is missing a corresponding patch on AccessTokenFactory.

Reply:
I've updated the arguments there